apps/ejabberd/src/ejabberd_sm.erl

```diff
@@ -202,7 +215,8 @@ get_user_resources(User, Server) ->
 
 -spec get_session_ip(User :: ejabberd:user(),
```

Comment:
For me the syntax with `when` is less readable but it's just personal preference. If nobody agrees with me, then I'll change it wherever I spot it during last fixes.